stetho/src/test/java/com/facebook/stetho/inspector/elements/android/ViewDescriptorTest.java

```diff
@@ -0,0 +1,54 @@
+package com.facebook.stetho.inspector.elements.android;
+
+import android.app.Activity;
+import android.os.Build;
+import android.widget.TextView;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.robolectric.Robolectric;
+import org.robolectric.RobolectricTestRunner;
+import org.robolectric.annotation.Config;
+
+import static org.mockito.Matchers.anyObject;
+import static org.mockito.Matchers.anyString;
+import static org.mockito.Mockito.*;
+
+@Config(emulateSdk = Build.VERSION_CODES.JELLY_BEAN)
+@RunWith(RobolectricTestRunner.class)
+public class ViewDescriptorTest {
+
+  private final MethodInvoker mMethodInvoker = mock(MethodInvoker.class);
+  private final ViewDescriptor mDescriptor = new ViewDescriptor(mMethodInvoker);
+  private final TextView mTextView = new TextView(Robolectric.setupActivity(Activity.class));
```

Comment:
let's also test something other than `TextView` ... like create a `CheckBox` and then do a `setAttributesAsText(mCheckBox, "checked=\"true\"")`